lib/vyasa/medium/voice.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Vyasa.Medium.Voice do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Vyasa.Written.{Source, Chapter}
+  alias Vyasa.Medium.{Event, Video, Track}
+
+  @primary_key {:id, Ecto.UUID, autogenerate: false}
+  schema "voices" do
+    field :lang, :string
+    field :title, :string
+    field :duration, :integer
+    field :file_path, :string, virtual: true
+
+    embeds_one :prop, VoiceProperties do
```

Comment:
naming: props should metadata

so far I've been hardcoding stub values for things like "artists" 


solve this before the pushing to merge

Reply:
donez